purchase_cancel_qty/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Purchase Cancel Qty
+===================
+
+Allow purchase users to define cancelled quantity on purchase order lines.
+* Quantity field becomes readonly when a PO is confirmed and its value is stored in an other field
```

Comment:
Add a blank line above, otherwise this is not correctly displayed.

Reply:
Indeed, thanks